cirq-google/cirq_google/engine/engine.py

```diff
@@ -877,3 +884,30 @@ def get_engine_calibration(
     May return None if no calibration metrics exist for the device.
     """
     return get_engine(project_id).get_processor(processor_id).get_current_calibration()
+
+
+def get_engine_sampler(processor_id: str, project_id: Optional[str] = None) -> cirq.Sampler:
+    """Get an EngineSampler assuming some sensible defaults.
+
+    This uses the environment variable GOOGLE_CLOUD_PROJECT for the Engine
+    project_id, unless set explicitly.
+
+    Args:
+        processor_id: Engine processor ID (from Cloud console or
+            ``Engine.list_processors``).
+        gate_set_name: One of ['sqrt_iswap', 'sycamore'].
+            See `cirq_google.NAMED_GATESETS`.
+        project_id: Optional explicit Google Cloud project id. Otherwise,
+            this defaults to the environment variable GOOGLE_CLOUD_PROJECT.
+            By using an environment variable, you can avoid hard-coding
+            personal project IDs in shared code.
+
+    Returns:
+        A `QuantumEngineSampler` instance.
```

Comment:
nit: I'd suggest removing this, since it doesn't add anything beyond the return type annotation. If you want to leave it, can just say it returns a sampler.

Reply:
Done.